docs/databases/postgres.md

```diff
@@ -4,4 +4,118 @@ title: Using PostgreSQL with Graphback
 sidebar_label: PostgreSQL
 ---
 
-## TODO
\ No newline at end of file
+[PostgreSQL](https://www.postgresql.org/) is a powerful, open source object-relational database system with over 30 years of active development that has earned it a strong reputation for reliability, feature robustness, and performance. [Graphback Knex Provider](https://www.npmjs.com/package/@graphback/runtime-knex) package provides instant integration to your PostgreSQL database, giving you a full implementation of the [CRUD API](../crud/introduction.md). 
+
+Additionally, Graphback comes with the [GraphQL Migrations](../graphql-migrations/intro.md) package that can be used to perform migrations against your database using the business models defined in the GraphQL Schema. 
```

Comment:
Technically, they are separate packages that work well together. But it need to be installed separately as `graphql-migrations` to be used, unless you are using a template.

Reply:
I see what you mean. How would you re-phrase this to avoid any confusion?

Reply:
How about:

Additionally you can use GraphQL Migrations to perform database migrations using the models defined in the GraphQL schema created by Graphback.

Reply:
Sold!